mvn_setup/tasks-parent/tasks-ui/src/core/TaskEdit/TaskEditDialog.tsx

```diff
@@ -3,9 +3,9 @@ import { Box, Stack, List, Dialog, Button, Grid } from '@mui/material';
 import SecurityIcon from '@mui/icons-material/Security';
 import { FormattedMessage } from 'react-intl';
 import TaskClient from '@taskclient';
-import { DatePicker } from '../../DatePicker/DatePicker';
-import { StyledFullScreenDialog } from '../StyledFullScreenDialog';
-import Fields from './EditTaskFields';
+import { DatePicker } from '../DatePicker/DatePicker';
```

Comment:
Actually, this import is incorrect. In the DatePicker folder, we have an `index.ts`, and we should be importing via the index.ts.

We are using the `index.ts` to define what things are public and what things are private.  If things are not exported via the `index.ts`, they are not meant to be used outside of that folder.